crates/client-api/src/routes/identity.rs

```diff
@@ -199,39 +197,39 @@ pub struct ValidateTokenParams {
 
 pub async fn validate_token(
     Path(ValidateTokenParams { identity }): Path<ValidateTokenParams>,
-    auth: SpacetimeAuthHeader,
+    SpacetimeAuthRequired(auth): SpacetimeAuthRequired,
```

Comment:
nit: this one ends up doing a `auth.identity != identity` check anyway, so would anonymous auth be fine like the others?

Reply:
I think given we're validating the token - it makes sense to require a token to validate against. If you don't pass a token to validate_token it was probably a mistake.